pallets/summary/src/challenge.rs

```diff
@@ -134,8 +135,9 @@ fn can_challenge<T: Config>(
         return false
     }
 
-    let is_chosen_validator = AVN::<T>::is_primary(ocw_block_number, &this_validator.account_id)
-        .unwrap_or_else(|_| false);
+    let is_chosen_validator =
+        AVN::<T>::is_primary(OperationType::Ethereum, &this_validator.account_id)
```

Comment:
Should this be `AvN`? I can see in /summary/lib.rs we use `AvN` when computing the primary. Is that for a different purpose?

Reply:
Should be Avn, yes.